_example/echo.proto

```diff
@@ -10,7 +10,7 @@ package echo;
 
 import "github.com/TuneLab/go-genproto/googleapis/api/serviceconfig/annotations.proto";
 
-service Echo {
+service Bounce_Echo {
```

Comment:
Why this name? Seems like an odd name. If it is to show how the folder name will change then it probably should have a comment :).